docs/kf.dev/content/en/docs/getting-started/install/_index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Install Kf"
+linkTitle: "Basic Kf Installation"
+weight: 10
+description: >
+  Learn how to use the `kf` CLI to create a new Kubernetes cluster with Kf
+  installed.
+---
+
+[for-operators]: /docs/operators
+Follow these instructions to quickly configure and deploy a basic Kf
+installation on a new Google Kubernetes Engine (GKE) cluster. For a more
+comprehensive installation guide for system operators/admins, please see the
+[Operators Guide][for-operators].
+
+## Prerequisites
+
+### Google Cloud
+[gke]: https://cloud.google.com/kubernetes-engine/
+[free]: https://cloud.google.com/free
+[new-project]: https://cloud.google.com/resource-manager/docs/creating-managing-projects
+These instructions assume Kf will be installed onto [Google Kubernetes Engine
+(GKE)][gke] in a Google Cloud Platform (GCP) project. If you do not have a GCP
+project, you can sign up for the [free tier][free].
+
+If you do not have an existing GCP project, create a new one by following [these
```

Comment:
The installer will also create a new project and sync billing for you. It might be worth mentioning that you have to setup billing.

Reply:
Oh nice! I'll replace this with instructions for billing, then.